### DISCUSSION TITLE
how to calculate server requirement for 500 concurrent user

### DISCUSSION THREAD

**Discussion by ahmadalwa** [mynaparrot/plugNmeet-server#578] [General]
how to calculate server requirement for 500 concurrent user

**Comment by yousofkakhki** [comment]
hi @ahmadalwa 
did you find any response?

**Comment by yousofkakhki** [comment]
Hi there. 
i wanted to actually host a webinar consisting of 1000 concurrent users. a teacher normally share webcam and mic to other 999 students. students should have the ability to raise hand and ask question once in a while with mic and webcam.
i have noticed that a room is only served through one livekit instance.
how to achieve this. i got some errors with nats and plugnmeet when load testing more than 400 users on pretty powerful machine. 48 core vds with 96 gb of ram and even the resources weren't  exhausted.
